src/components/NewComps/AffiliateForm/AffiliateForm.module.scss

```
@@ -0,0 +1,147 @@
+  }
+
+  :global(.iti--separate-dial-code .iti__selected-dial-code) {
+    font-size: 14px;
+    color: var(--col-slate-grey);
+  }
+
+  :global(.iti__flag-container) {
+    padding: 0;
+  }
+
+  :global(.iti--separate-dial-code .iti__selected-flag) {
```

Stylelint (17.13.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)